### ISSUE TITLE
Email To Customer on user Assignment

### ISSUE THREAD

**Issue by VacationTeam** [freescout-helpdesk/freescout#1697]
Whenever an email is assigned to a user, either manually or by work flow
FreeScout is sending a copy of that message to the customer
Logs show email to customer.  But I do not have this setup as an action.  
Some emails are doing this in an endless loop where free scout adds and new blank line, and sends again, over and over

![image](https://user-images.githubusercontent.com/90206057/152197032-9a5cf5d5-b8bc-4a8a-8921-6355b4f8f094.png)

**Comment by VacationTeam** [comment]
This job keeps queue'ing and executing as well
![image](https://user-images.githubusercontent.com/90206057/152197899-b93dc8f5-afb4-46ee-b789-8c3d3ab18b21.png)

**Comment by VacationTeam** [comment]
Just to note.
Clear Cache, Changing permissions nothing cleared this jobs from generating.
I tried the cancel button, but jobs kept generating.
I restarted the server and it has stopped.
Having trouble pinpointing the underlying issue

**Comment by freescout-helpdesk** [comment]
Check Logs.

**Comment by bladerunner2020** [comment]
We also have this issue.

**Comment by VacationTeam** [comment]
> Check Logs.

They are non-descriptive.
I have logs showing email to customer.  Doesn't say what triggered it.

I have an app log, but the time stamp is an hour later, with a database error it seems.
Can you advise if this has something to do with it?

In addition. I instealled the mentions module. Mentioning someone, (@) causes this same thing to happen where Scout sends endless emails to the customer.
It adds a new blank line on each email, over and over and over again.
I can only stop it by killing all processes and turning off email notifications for everything for users.

This was a fresh install of the software and modules

Thank you
![log2](https://user-images.githubusercontent.com/90206057/152682063-94d81246-a0c0-4ae4-9af1-55a6ef4dbfba.jpg)
![Log1](https://user-images.githubusercontent.com/90206057/152682065-8103d252-aed2-4659-91e7-75c71af5a48f.jpg)

**Comment by freescout-helpdesk** [comment]
What is your PHP and FreeScout version? Do you have any non-official modules installed? Are you using docker?

**Comment by VacationTeam** [comment]
> What is your PHP and FreeScout version? Do you have any non-official modules installed? Are you using docker?

No non-official modules, all purchased from the FreeScout website.
Using: 
Custom Folders, Inbox, Mentions, Tax, Workflows.

Using an AWS Instance. (Through Cloudways). 
Freescout - 1.7.30
Php - PHP 7.4.27

I see the system is telling me there is an update to 1.8.3 Today.
Let me know if I should update.

Thank you

**Comment by bladerunner2020** [comment]
In our case we have one agent and his email is the same as Helpdesk email. When a ticket is assigned, the notification  is sent to agent and as it's the same as mailbox email, it's received by FreeScout. FreeScout then thinks that it was reply of agent and send the reply to the customer.

We changed the agent's email and solved this issue.

**Comment by VacationTeam** [comment]
Solved.
Some how sent emails were routing into my "Google Apps" inbox.
Scout would then pull that notification to user, attach it to the origonal message, then message the customer thinking it was a reply.
And so the loop starts over and over and over.

I fixed sent emails showing up in the Google Inbox and problem has been solved.
Also, make sure to not use any gmail filters with Never Mark Spam as an option, it causes emails from sent to go into the inbox.
